docs/REST API/rest-api.md

```diff
@@ -5378,7 +5216,7 @@ The summary can optionally be filtered by team ID.
 
 _Available in Fleet Premium_
 
-`PATCH /api/v1/fleet/mdm/apple/setup`
+`PATCH /api/v1/fleet/setup_experience`
```

Comment:
@rachaelshaw Hey, just a heads-up that in a call with Noah and Marko, we opted to use this endpoint for the UI to also set the `enable_release_device_manually` option as part of [this story](https://github.com/fleetdm/fleet/issues/9147).

The reason this is required (vs using the existing `PATCH /config` or Apply team specs endpoints, which will also support this option but for `fleetctl apply`) is due to the permissions difference - only admins can use the patch config/team specs endpoints, while in the UI, maintainers also need to be able to enable this setting.

Do you think that makes sense? The title of this section would need to be changed accordingly (line 5215 above) as currently it mentions "turn on end user authentication" specifically. And the `enable_release_device_manually` parameter (boolean) would need to be added.

Reply:
@noahtalerman @marko-lisica did this API endpoint end up getting a new parameter?

I saw this issue was closed and the API docs merged in [here](https://github.com/fleetdm/fleet/pull/17118), so I'm assuming this went a different direction— just want to check that there weren't other changes that didn't end up being documented.

Reply:
I'm not sure. 

> we opted to use this endpoint for the UI to also set the enable_release_device_manually option as part of https://github.com/fleetdm/fleet/issues/9147.

Hey @mna, did we end up making this change? If so, it sounds like we're missing this in the docs.

Reply:
@noahtalerman @rachaelshaw Yes we did make that change.